### DISCUSSION TITLE
Config file paramter in Gen_Ens_Prod conf file

### DISCUSSION THREAD

**Discussion by BinbinZhou-NOAA** [dtcenter/METplus#1259] [Configuration] [answered]
As George pionted me the conf file for user case of GenEnsProd 
 https://github.com/dtcenter/METplus/blob/7094e0a613d354d894119900327837b6da143644/parm/use_cases/met_tool_wrapper/GenEnsProd/GenEnsProd.conf#L3 

I can not find the GEN_ENS_PROD_CONFIG_FILE variale in this use case conf file.   How to define the Gen_Ens_Prod  config file and its path in a  conf file?  

Binbin

**Comment by georgemccabe** [comment, marked as answer]
Hi @BinbinZhou-NOAA, GEN_ENS_PROD_CONFIG_FILE is now optional. The default value is the wrapped MET config file for the tool found in parm/met_config, i.e. GenEnsProdConfig_wrapped. It is rare that you will need to change this to another file because you can set any unsupported config overrides in the wrapped version using GEN_ENS_PROD_MET_CONFIG_OVERRIDES (see https://metplus.readthedocs.io/en/latest/Users_Guide/systemconfiguration.html#overriding-unsupported-met-configuration-variables).

If you really need to use another MET config file to run, then you can override the value of GEN_ENS_PROD_CONFIG_FILE to the path to your MET config file. However, as support for new configurations are added to the wrapped version in the repository, you will have to make updates to your MET config file. This is why we recommend always using the wrapped MET config file that is provided and set any overrides you need in your METplus config file.

**Reply by BinbinZhou-NOAA** [reply]
George,

  Got it! 

Thanks!

Binbin
